Freshli.Web/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -27,6 +27,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("uuid");
 
                     b.Property<string>("Email")
+                        .IsRequired()
```

Comment:
Ah... I bet this is what the migration actually picked up. Still kind of cool. :)